lib/parse-args.js

```diff
@@ -158,8 +158,17 @@ function buildYargs (withCommands = false) {
       describe: 'supplying --merge-async will merge all v8 coverage reports asynchronously and incrementally. ' +
         'This is to avoid OOM issues with Node.js runtime.'
     })
+    .options('print-config', {
+      default: false,
+      type: 'boolean',
+      describe: 'Print the derived configuration between command line parameters and loaded configuration file'
+    })
+    .options('print-config-format', {
+      default: 'text',
```

Comment:
You can use choices for this, to enforce `json` or `text`:

https://github.com/yargs/yargs/blob/main/docs/api.md#choiceskey-choices